clireporter/cli.go

```diff
@@ -75,7 +75,11 @@ func option(logger *logger) func(*taskrunner.Runtime) {
 				case *taskrunner.TaskStartedEvent:
 					logger.Write(task, "Started")
 				case *taskrunner.TaskCompletedEvent:
-					logger.Writef(task, "Completed (%0.2fs)", float64(event.Duration)/float64(time.Second))
+					if event.Duration == 0 {
```

Comment:
nit: maybe add a comment that it's the zero value since we never set the duration otherwise?

Reply:
not sure I 100% follow